package.json

```diff
@@ -115,6 +121,11 @@
     "@storybook/cli": "4.1.9",
     "@storybook/react": "4.1.9",
     "@storybook/storybook-deployer": "2.8.1",
+    "@types/jest": "24.0.19",
+    "@types/node": "12.11.1",
+    "@typescript-eslint/eslint-plugin": "2.4.0",
+    "@typescript-eslint/parser": "2.4.0",
+    "awesome-typescript-loader": "5.2.1",
```

Comment:
I saw different typescript loaders, this one and at least another one, are we sure its the best loader to use ? (is it well maintained or deprecated?)

Reply:
Yeah, both ts-loader and awesome-typescript-loader are well maintained and used by thousands of people. awesome-typescript loader was faster for me so I went with that.